pyartifactory/objects.py

```diff
@@ -784,17 +812,24 @@ def deploy(
         :param artifact_path: Path to file in Artifactory
         :param local_file_location: Location of the file to deploy
```

Comment:
Maybe update this documentation to indicate that it can also accept a folder?
Like this?
```suggestion
        Deploy an artifact or a folder to an Artifactory repository.
        If deploying a file, the artifact_path MUST end with the destination artifact name.
        :param local_file_location: Location of the file or folder to deploy
        :param artifact_path: Path to the artifact in Artifactory
```